cc_hooks_gentoo.py

```diff
@@ -824,7 +824,9 @@ def pre_configure_hook(self, *args, **kwargs):
     orig_enable_templating = self.cfg.enable_templating
     self.cfg.enable_templating = False
 
-    modify_all_opts(self.cfg, opts_changes)
+    modify_all_opts(self.cfg, opts_changes,
+        opts_to_skip=['builddependencies', 'dependencies', 'modluafooter',
```

Comment:
why the addition of these options to skip ?

Reply:
because the default option list has those (see `cc_hooks_common.py`) Should we edit it there?

Reply:
https://github.com/ComputeCanada/easybuild-computecanada-config/blob/4077e531881e9a8030c502564effc1354c22e4df/cc_hooks_common.py#L30

Reply:
ah, I see.